### PR TITLE
Made Hamburger Menu Accessible by Keyboard

### PR DESCRIPTION
**Pull request by vjcao**
Currently the hamburger menu in the app bar is not accessible through the use of a keyboard. When one presses 'Tab' when inside the menu, the focus exits the menu and moves on to the next button.  This PR addresses issue #804 and aims to fix this accessibility issue. 

**Current Hamburger Menu:**
Jan-31-2020 15-58-59

**This PR's Hamburger Menu:**
Jan-31-2020 16-03-20

### CODE REVIEW

**Comment by jonespm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ITS_Teaching_And_Learning/my-learning-analytics/pullRequest?prid=4926457&bid=16299115)

File: assets/src/containers/Course.js [hunk 1]
```diff
@@ -16,6 +16,16 @@ function Course (props) {
   const [loaded, error, courseInfo] = useCourseInfo(courseId)
   const [sideDrawerState, setSideDrawerState] = useState(false)
 
+  const toggleDrawer = (open) => event => {
```
Comment (pushyamig) [inline assets/src/containers/Course.js]: This can be separated to its own component and imported and used here

**Comment by pushyamig** [comment]
>This PR also changes how current views are displayed in the menu.
@vjcao Hamburger menu changes need to be pulled separately from the Current view indicator with start. Since it is difficult to say which changes are to what. 
Create a new issue and separate PR for those fixes.

File: assets/src/containers/Course.js (hunk 1 above)
Comment (vjcao) [reply assets/src/containers/Course.js]: This will be resolved later

**Comment by jonespm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ITS_Teaching_And_Learning/my-learning-analytics/pullRequest?prid=4926457&bid=16299115)

**Comment by pushyamig** [comment]
@vjcao can you please resolve the conflicts you have with this PR?

**Comment by pushyamig** [comment]
This PR is behaving as expected in Chrome but not in Firefox and safari
In Firefox when you Tab and enter on Hamburger icon it opens up the menu but from there if you cannot navigate to the respective view by (Tab and Enter)
Tabbing doesn't work at all in Safari.
Chrome seems to the most used browser from looking at the report and next comes to Safari. 
Screen Shot 2020-02-27 at 11 13 36 AM

I feel we should at least make this work in Safari.

**Comment by vjcao** [comment]
@pushyamig I have tested this PR on Chrome, Firefox, Safari, Edge, and Opera. There was actually nothing wrong with the code. It was macOS's fault. For the changes in this PR to work on Firefox and Safari, macOS users must go to System Preferences -> Keyboard -> Shortcuts and check 'Use keyboard navigation to move focus between controls.' After doing so, the hamburger menu performs as expected.

**Comment by pushyamig** [comment]
@vjcao Looks good. Can you also add all the info about enabling the accessibility features on Windows/Mac in the Myla wiki page that makes it easy for testing? Add the link to that wiki page (how to enable the accessibility option in different OS ) in the Issue so that way it will be easy for testing this issue

**Pull request merged**